package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-plugin-vuetify": "^1.1.0",
     "flush-promises": "^1.0.2",
     "jsdom": "^22.1.0",
+    "lerna": "^7.4.1",
```

Comment:
Maybe we should add a "version" (after line 2) to this repo as well, eh?

I'm thinking either 2.0.0 since it's the first significant change to this repo's top level, or maybe 3.0.0 to coincide with upcoming Vue changes?

Reply:
Yes makes sense. I think 3.0.0 to coincide

Reply:
Updated version to 3.0.0